### ISSUE TITLE
Error on TAM Montpellier network

### ISSUE THREAD

**Issue by brenard** [schildbach/public-transport-enabler#272]
Hello, 

Any itinary in Montpellier return the following error :  
![Screenshot_20190709-090749](https://user-images.githubusercontent.com/421049/60866647-2e8f1a80-a229-11e9-8f63-55acbb6004a1.jpg)

**Comment by schildbach** [comment]
Please don't post screenshots. Always post the full exception trace in textual form.

**Comment by brenard** [comment]
Ok, so you have to adjust the text in the app if you don't want screenshot because it's explicitly asked today. Related note : the URL to post issue cited in the app is incorrect (https://github.com/schildbach/public-transport-enabler/issuespublic-transport-enabler).

Following, the exception of this error in plain text :

```
java.lang.NullPointerException
com.google.common.base.Preconditions.checkNotNull(Preconditions.java:890)
de.schildbach.pte.dto.Style.parseColor(Style.java:108)
de.schildbach.pte.FranceSouthEastProvider.getLineStyle(FranceSouthEastProvider.java:65)
```

**Comment by schildbach** [comment]
Thanks. Regarding the message and URL, it's not our app. Please take that feedback to Transportr.

**Comment by brenard** [comment]
You are right ! An issue about it already exists on project Transportr, so [I just add a comment](https://github.com/grote/Transportr/issues/614#issuecomment-509749389).

**Comment by schildbach** [comment]
Should be fixed via 7f607160ca919bd68628d867f0a80972c7a471ae.
